pages/add/form/get-help-fields.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import validator from './validator';
 import Service from '../../../js/service';
 
-let HelpTypes = React.createClass({
-  getInitialState() {
-    return {
+class HelpTypes extends React.Component {
+  constructor() {
```

Comment:
@Pomax I've read some articles online saying if we are not using `props` to set `state` there's no need to pass `props` to `super()`. The `App` class in `routes.jsx` seems to be functioning normally without passing `props` to `super()`.